vector/src/main/java/im/vector/activity/RiotAppCompatActivity.java

```diff
@@ -18,10 +18,14 @@
 package im.vector.activity;
 
 import android.content.Context;
+import android.os.Bundle;
```

Comment:
this is strange to see only new imports. I think they are not used anymore. Remove them